exasol/toolbox/tools/replace_version.py

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
+from typing import List
+
+def replace_version(template: Path, version: str) -> None:
+    with open(template, "r+", encoding="utf8") as file:
+        lines = file.readlines()
+        _replace(lines, "exasol/python-toolbox/.github/", version)
+        file.seek(0)
+        file.writelines(lines)
+
+
+def _replace(lines: List[str], replace_filter: str, version: str) -> None:
+    for count, line in enumerate(lines):
+        if line.find(replace_filter) != -1:
+            if line.find("@") != -1:
+                lines[count] = line[0:line.index("@")+1] + version + "\n"
```

Comment:
if you have a loop(s) which select a subset of the iterated items, you usually can simplify the code by creating an iterable over the items you want to take action on. e.g.:

```python
    filtered_lines = ((index, line) for index, line in enumerate(lines) if line.find(replace_filter) != -1)
    filtered_liens = [(index, line) for index, line in filtered_lines if line.find("@") != =1]
    for index, line in filtered_lines:
        lines[index] = line[0:line.index("@")+1] + version + "\n"
```